.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,24 +1,26 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'status: needs triage, type: bug'
+assignees: ''
+
+---
+
 <!--
 If you encounter a bug or you have a support question, please try to fill out some of the information below.
```

Comment:
For some reason it won't let me make a suggestion here, but here's what I suggest:
```
Before opening an issue, please take a look at the [troubleshooting guide](https://haskell-language-server.readthedocs.io/en/latest/troubleshooting.html). This explains some common issues and will also help you to find the information that the issue template asks for.

When filing an issue, please fill out as much of the information below as you can. This helps us to debug your issue, but is not required!
```

Reply:
done!